docs/load-balancer/sticky.md

```diff
@@ -0,0 +1,29 @@
+# Sticky Load Balancing
+
+The sticky load balancer selects a single service instance and keeps using it until it fails.
+Then, it selects the next one.
+
+It is possible to configure a back off time to specify for how long a service instance that failed should not be retried.
+
+Precisely, the load balancer works as follows:
+
+* if the previously selected service instance has not failed, and is still available, return it
+* else if exists, return the first available service instance that has no recorded failure
+* else, find the available instance that has failed the longest time ago, and
```

Comment:
```suggestion
* otherwise, find the available instance that was the first to record a failure, and
```

Reply:
that's not technically true. 
Let's assume we have services A and B.
A failed first, then B failed while backoff time had passed for A. So we used A. Then A failed.
At this point we don't select A.

Reply:
I reworded it a bit